pytest_operator/plugin.py.orig

```diff
@@ -0,0 +1,492 @@
+import asyncio
```

Comment:
This file looks like it was checked in by accident. Remove it?

Reply:
Ah, yep.  Thanks for the catch.